src/content/docs/en/guides/images.mdx

```diff
@@ -159,6 +159,37 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
 <ReadMore>See details about [the `<Picture />` component properties](/en/reference/modules/astro-assets/#picture-properties) in the `astro:assets` reference.</ReadMore>
 
+### Inline `.svg` files in the HTML
+
+<p><Since v="5.0.0" /></p>
+
+To inline local `.svg` files directly in your HTML, SVG files can be imported and used like any other [Astro components](/en/basics/astro-components/#component-based-design).
+
+```astro
+---
+import Logo from '../assets/logo.svg';
+---
+
+<Logo />
+```
+
+You can set attributes such as `width`, `height`, `fill`, `stroke`, and other common SVG attributes on the imported component. These attributes will automatically be applied to the underlying `<svg>` element. Properties passed to the component will override duplicate properties that exist in the original `.svg` file.
+
+
+```astro
+<Logo width={64} height={64} fill="currentColor" />
+```
+
+
+As a convenience, SVG components accept a `size` property. `size` is a shorthand which sets both the `width` and `height` properties to the same value.
+
+**Example:**
```

Comment:
```suggestion
The following example uses Astro's `size` property to set an equal width and height of 48 instead of setting the HTML `<svg>` attributes of `width` and `height` separately:
```